dev/group_vars/apim.yml

```diff
@@ -111,16 +98,31 @@ wso2_mb_store_db_username: wso2carbon
 wso2_mb_store_db_password: wso2carbon
 wso2_mb_store_db_validation_query: SELECT 1
 
+wso2um_db_driver: org.h2.Driver
+wso2um_db_url: jdbc:h2:repository/database/WSO2UM_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000;AUTO_SERVER=TRUE
+wso2um_db_username: wso2carbon
+wso2um_db_password: wso2carbon
+wso2um_db_validation_query: SELECT 1
+
+wso2reg_db_driver: org.h2.Driver
+wso2reg_db_url: jdbc:h2:repository/database/REGISTRY_DB;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=60000;AUTO_SERVER=TRUE
+wso2reg_db_username: wso2carbon
+wso2reg_db_password: wso2carbon
+wso2reg_db_validation_query: SELECT 1
+
 # Key Store and Trust Store configuration
 key_store_location: repository/resources/security/wso2carbon.jks
 key_store_password: wso2carbon
 key_store_key_password: wso2carbon
+key_store_key_alias: wso2carbon
 trust_store_location: repository/resources/security/client-truststore.jks
 trust_store_password: wso2carbon
 
 # The KeyStore which is used for encrypting/decrypting internal data. This block is read by Carbon Crypto Service
 internal_keystore_location: ${carbon.home}/repository/resources/security/wso2carbon.jks
 internal_keystore_key_password: wso2carbon
+internal_keystore_password: wso2carbon
+internal_keystore_key_alias: wso2carbon
 
 # User Manager configurations
 datasource: jdbc/WSO2CarbonDB
```

Comment:
Shall we remove this?

Reply:
Done https://github.com/wso2/ansible-apim/pull/49/commits/6464f58613fc352dd7b9fcc50d0bb1c1f9d8a6e1